src/components/shared/Validations.tsx

```diff
@@ -86,3 +88,15 @@ export const Validations: React.FunctionComponent<IValidations> = ({
     </>
   );
 };
+
+export const Format: React.FunctionComponent<{ schemaNode: JSONSchema4 }> = ({ schemaNode }) => {
```

Comment:
schemaNode !== JSONSchema4

Reply:
My fault, this parameter should actually be named `nodeSchema`, since it accepts a schema of a node. `SchemaNode` probably doesn't not apply here though, beacuse it only contains 2 elements: `id` and `?title` and we are looking for a validation

Reply:
let's call the prop `schema` then.
I think that's how we usually call it.

`SchemaNode` does have more properties. Would only need to include `format` (could be done in a similar way to `enum`).
In any case, using `schema` here is alright.